domain/src/main/java/org/oppia/android/domain/question/QuestionAssessmentProgressController.kt

```diff
@@ -24,8 +24,12 @@ import javax.inject.Inject
 import javax.inject.Singleton
 import kotlin.concurrent.withLock
 
-private const val CURRENT_QUESTION_DATA_PROVIDER_ID = "CurrentQuestionDataProvider"
+private const val CREATE_CURRENT_QUESTION_DATA_PROVIDER_ID =
+  "create_current_question_data_provider_id"
+private const val BEGIN_QUESTION_TRAINING_SESSION_PROVIDER_ID =
+  "begin_question_training_session_provider_id"
 private const val EMPTY_QUESTIONS_LIST_DATA_PROVIDER_ID = "EmptyQuestionsListDataProvider"
```

Comment:
Nit fix:
`private const val EMPTY_QUESTIONS_LIST_DATA_PROVIDER_ID = "empty_questions_list_data_provider_id"`

Reply:
Fixed

Reply:
Thanks for doing this correctly. I appreciate it. @misterjedu